cmd/filters/filters.go

```diff
@@ -0,0 +1,39 @@
+package filters
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/k8sgpt-ai/k8sgpt/pkg/analyzer"
+	"github.com/spf13/cobra"
+)
+
+var FiltersCmd = &cobra.Command{
+	Use:     "filters",
+	Aliases: []string{"filters"},
+	Short:   "Manage filters for analyzing Kubernetes resources",
+	Long: `The filters command allows you to manage filters that are used to analyze Kubernetes resources.
+	You can list available filters to analyze resources.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			cmd.Help()
+			return
+		}
+	},
+}
+
+var listCmd = &cobra.Command{
```

Comment:
Can you please add this into it's own file? More for consistency than anything else really!

Reply:
Done